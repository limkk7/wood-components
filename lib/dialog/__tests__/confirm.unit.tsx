import * as renderer from "react-test-renderer";
import * as React from "react";
import { confirm, Button } from "../../wood-components";

describe("测试 dialog", () => {
  it("confirm", () => {
    const json = renderer
      .create(
        <Button
          onClick={() =>
            confirm(
              "confirm",
              () => {
                console.log("hah");
              },
              () => {
                console.log("oh no");
              }
            )
          }
        >
          show confirm
        </Button>
      )
      .toJSON();
    expect(json).toMatchSnapshot();
  });
});
