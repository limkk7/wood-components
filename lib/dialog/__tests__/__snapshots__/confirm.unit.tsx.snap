// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 dialog confirm 1`] = `
<button
  className="w-ui-button"
  onClick={[Function]}
>
  show confirm
</button>
`;
